client/nginx.go

```diff
@@ -654,3 +733,12 @@ func (client *NginxClient) getUpstreams() (*Upstreams, error) {
 	}
 	return &upstreams, nil
 }
+
+func (client *NginxClient) getStreamUpstreams() (*StreamUpstreams, error) {
+	var upstreams StreamUpstreams
+	err := client.get("stream/upstreams", &upstreams)
+	if err != nil {
```

Comment:
If the stream {} block is not present in the NGINX Plus configuration, NGINX Plus will return 404 "stream not configured (StreamNotConfigured)" for any /stream API calls. If that is the case, our client must recognize that and not fail!

Reply:
done